File: Praktikum 3 Python/PRAK304-2510817220002-AishaZwarintyanda.py
angka = int(input("Masukkan Nilai: "))

if angka == 0:
    print("Nol")
elif angka > 0 and angka < 10:
    print("Satuan")
elif angka > 10 and angka < 20:
    print("Belasan")
elif angka > 19 and angka <99:
    print("Puluhan")
elif angka == 10:
    print("Puluhan")
else:
    print("Anda menginput melebihi limit bilangan")

File: Praktikum 3 C/PRAK304-2510817220002-AishaZwarintyanda.c
#include <stdio.h>

int main()
{
    int n;
    char *kata;
    printf("Masukkan nilai: ");
    scanf("%d", &n);

    if (n == 0)
    {
        kata = "Nol";
    }
    else if (n >= 1 && n <= 9)
    {
        kata = "Satuan";
    }
    else if (n == 10 || (n >= 21 && n <= 99))
    {
        kata = "Puluhan";
    }
    else if (n >= 11 && n <= 19)
    {
        kata = "Belasan";
    }
    else
    {
        kata = "Anda Menginput Melebihi Limit Bilangan";
    }

    printf("%s\n", kata);

    return 0;
}